gcloud/rest/taskqueue/manager.py

```diff
@@ -104,13 +77,13 @@ def find_and_process_work(self):
                 tasks = [tasks[0]]
 
                 for task in to_burn:
-                    log.info('burning task %s', task.get('id'))
+                    log.info('burning task %s', task.get('name'))
```

Comment:
I assume this means that the task metadata has changed in the update? If so, would this impact the clients of this lib (i.e. `gc-*` services) or does separation of concern cover this?

Reply:
Indeed. _Theoretically_, the point of this lib in the first place was to enforce the separation of concerns, but we'll have to double-check when we bump this dependency in the `gc-*` services.